templates/admin.php

```diff
@@ -90,7 +90,8 @@
 					</p>
 				<?php elseif ($attribute['type'] === 'line' && !isset($attribute['global'])): ?>
 					<p>
-						<input data-key="<?php p($key)?>" name="<?php p($key) ?>" value="<?php p($_['config']['general-'.$key] ?? '') ?>" type="text" <?php if (isset($attribute['required']) && $attribute['required'] === true): ?>class="required"<?php endif;?> placeholder="<?php p($attribute['text']) ?>"/>
+						<label for="<?php p($key)?>"><?php p($attribute['text']) ?></label><br/>
+						<input data-key="<?php p($key)?>" name="<?php p($key) ?>" value="<?php p($_['config']['general-'.$key] ?? '') ?>" type="text" <?php if (isset($attribute['required']) && $attribute['required'] === true): ?>class="required"<?php endif;?> />
```

Comment:
```suggestion
						<input data-key="<?php p($key)?>" id="<?php p($key) ?>" name="<?php p($key) ?>"  value="<?php p($_['config']['general-'.$key] ?? '') ?>" type="text" <?php if (isset($attribute['required']) && $attribute['required'] === true): ?>class="required"<?php endif;?> />
```

The label needs to point to an `id` https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#for